src/css/variables.less

```diff
@@ -31,6 +31,7 @@
  * ==============
  */
 @base-font-size: 14px;
+@base-line-height: 1.1428571429;
```

Comment:
Yep, 1.1428571429 equals 16px as it is relative to the font size (16/14=1.142...). So we should end up with 48px height in total.